### ISSUE TITLE
Similar project

### ISSUE THREAD

**Issue by axelkar** [igor-prusov/dts-lsp#35]
Hey! I noticed your project on crates.io! I have been working on a project [`dt-tools`](https://github.com/axelkar/dt-tools) but have decided that it's way too hard to parse the DTS-v1 format for IDE purposes. Feel free to take ideas from my code and the README!

**Comment by igor-prusov** [comment]
Hi, thanks for reaching out! I'll make sure to take a closer look at your project. Linting and bindings check look like really usefull features.

**Comment by axelkar** [comment]
https://github.com/Schottkyc137/ginko/issues/17 Here's more details on dt-tools

**Comment by igor-prusov** [comment]
@axelkar Thanks, I'll try to read that thread this week. It would be nice to join efforts, even though my project is built around tree-sitter and not a custom parser. I am still not sure which approach will work best in a long run.
